modules/button/react/lib/IconButton.tsx

```diff
@@ -52,7 +52,7 @@ const IconButton: ButtonOrAnchorComponent<
 > = ({
   theme = useTheme(),
   variant = IconButtonVariant.Circle,
-  size = 'medium',
+  size = variant === IconButtonVariant.ToolbarSquare ? 'small' : 'medium',
```

Comment:
Same here. If this has different defaults, it should be a separate component. Thoughts on pulling this out of `IconButton` and adding a `ToolbarIconButton`?